hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestFromClientSide.java

```diff
@@ -1156,20 +1156,28 @@ public void testSingleRowMultipleFamily() throws Exception {
   }
 
   @Test
-  public void testNull() throws Exception {
-    final TableName tableName = TableName.valueOf(name.getMethodName());
-
+  public void testNull_TableName() {
```

Comment:
testNullTableName? I thought we should keep use Camel-Case for method name.

Reply:
Sure. I'll fix them.